tensorflow_addons/image/testing_imageops.ipynb

```diff
@@ -0,0 +1,728 @@
+{
```

Comment:
<p>Likewise remove the logging info</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/tensorflow/addons/pull/368/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>